pkg/session_recording/recordings.jsx

```diff
@@ -249,7 +254,7 @@
                     <div className="row">
                         <div className="col-md-12">
                             <ol className="breadcrumb">
-                                <li><a>Session Recording</a></li>
+                                <li><a onClick={this.goBackToList}>Session Recording</a></li>
```

Comment:
Okay, I've fixed it for now, it seems to be fine, though I am not sure how to test properly if data in terminal displayed correctly while resizing. So, it seems onWindowResize call was unnecessary. Bug was because at the moment onWindowResize was called there is not refs - because this lib bounds resize on window ( globally ), so resize event happens before React renders elements. What do you think?